src/main/webapp/app/exercises/shared/exercise-update-warning/exercise-update-warning.service.ts

```diff
@@ -84,9 +84,9 @@ export class ExerciseUpdateWarningService {
      * @param backupInstruction original not changed instruction
      */
     checkInstruction(instruction: GradingInstruction, backupInstruction: GradingInstruction): void {
-        // instruction credits changed?
+        // structured grading instruction credits changed?
```

Comment:
```suggestion
        // checking whether structured grading instruction credits changed
```

Reply:
I can probably update this line inside of  PR I am working on, if there is no other major problem

Reply:
Sure, it's anyways just a minor things - maybe even just a matter of taste 👍